app/models/event.rb

```diff
@@ -0,0 +1,11 @@
+class Event < ApplicationRecord
```

Comment:
Let us put validations at least on the presence of event name and date. Right now I can create an empty event.